addon/components/power-calendar-multiple/days.js

```diff
@@ -5,5 +5,10 @@ export default DaysComponent.extend({
   dayIsSelected(dayMoment, calendar = this.get('calendar')) {
     let selected = calendar.selected || [];
     return selected.some((d) => dayMoment.isSame(d, 'day'));
+  },
+
+  dayIsDisabled(dayMoment) {
+    let numSelected = this.get('calendar.selected.length');
```

Comment:
If I read this right, if `maxLength` is 3, when you've selected 3 items then all the days become disabled, except those that are already selected (so you can unselect one and select another). Correct?

I think that you have to account for the posibility that `this.get('calendar.selected.length');` can be null. Also `maxLength` will be null if you don't  pass one, so this comparison might be broken.